src/ncp/ncp_spinel.hpp

```diff
@@ -203,13 +203,17 @@ class NcpSpinel
 
     static otDeviceRole SpinelRoleToDeviceRole(spinel_net_role_t aRole);
 
-    void HandleNotification(const uint8_t *aFrame, uint16_t aLength);
-    void HandleResponse(spinel_tid_t aTid, const uint8_t *aFrame, uint16_t aLength);
-    void HandleValueIs(spinel_prop_key_t aKey, const uint8_t *aBuffer, uint16_t aLength);
-    void HandleResponseForCommand(spinel_tid_t aTid, otError aError);
+    void      HandleNotification(const uint8_t *aFrame, uint16_t aLength);
+    void      HandleResponse(spinel_tid_t aTid, const uint8_t *aFrame, uint16_t aLength);
+    void      HandleValueIs(spinel_prop_key_t aKey, const uint8_t *aBuffer, uint16_t aLength);
+    otbrError HandleResponseForPropSet(spinel_tid_t      aTid,
+                                       spinel_prop_key_t aKey,
+                                       const uint8_t    *aData,
+                                       uint16_t          aLength);
 
     spinel_tid_t GetNextTid(void);
     void         FreeTid(spinel_tid_t tid) { mCmdTidsInUse &= ~(1 << tid); }
+    void         FreeTidTableItem(spinel_tid_t aTid);
```

Comment:
An idea/suggestion: Can we consider removing `FreeTid()` and always use the `FreeTidTableItem()`? (basically having one way to track pending TIDs)? 

Maybe something to consider in its own PR for future?

Reply:
Updated! This is a small change.